README.md

```diff
@@ -12,16 +12,33 @@ Licensed under the MIT License.
 [![Chrome Web Store](https://img.shields.io/chrome-web-store/stars/pbjjkligggfmakdaogkfomddhfmpjeni.svg)](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni/reviews)
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=microsoft/accessibility-insights-web)](https://dependabot.com)
 
-Accessibility Insights for Web is a browser extension for Google Chrome and the new Microsoft Edge, used for assessing the accessibility of web sites and web applications.
+Two projects are built from this repository:
 
-### Running the extension
+-   Accessibility Insights for Web is a browser extension for Google Chrome and the new Microsoft Edge, used for assessing the accessibility of web sites and web applications.
+-   Accessibility Insights for Android is a cross-platform desktop tool used for testing accessibility of Android applications.
+
+### Running Accessibility Insights for Web
 
 You can install the extension from one of the following links
 
 -   ![Canary Logo](./src/icons/brand/red/brand-red-16px.png) [Canary](https://chrome.google.com/webstore/detail/hbcplehnakffdldhldncjlnbpfgogbem) (released continuously)
 -   ![Insider Logo](./src/icons/brand/violet/brand-violet-16px.png) [Insider](https://chrome.google.com/webstore/detail/nnmjfbmebeckhpejobgjjjnchlljiagp) (on feature completion)
 -   ![Production Logo](./src/icons/brand/blue/brand-blue-16px.png) [Production](https://chrome.google.com/webstore/detail/pbjjkligggfmakdaogkfomddhfmpjeni) (after validation in Insider)
 
+### Running Accessibility Insights for Android
+
+You can install Accessibility Insights for Android from one of the following links. The application requires an APK which you can download [here](https://accessibilityinsights.io/en/downloads).
+
+-   MacOS ([Canary](https://aka.ms/accessibility-insights-for-android/downloads/CanaryMacOS), [Insider](https://aka.ms/accessibility-insights-for-android/downloads/InsiderMacOS), [Production](https://aka.ms/accessibility-insights-for-android/downloads/MacOS))
```

Comment:
@ferBonnin I have added two duplicate formats, one with bullets and one with a table. Which do you like better? You can see it [here](https://github.com/microsoft/accessibility-insights-web/blob/1e5d423653196b7c00dce2737d4fe9ac67ff7533/README.md).

Reply:
thanks! I like the first one, bullet points better: visually, it looks cleaner